.markdownlinkcheck.json

```diff
@@ -1,7 +1,8 @@
 {
-    "ignorePatterns": [{
-        "pattern": "^https://calendar.google.com/calendar"
-    }],
+    "ignorePatterns": [
+        { "pattern": "^https://calendar.google.com/calendar" },
+        { "pattern": "^../reference/" }
```

Comment:
I was not ambitious enough to check if this JSON accepts comments, but we should ignore checking links to these pages since they are generated just-in-time for Netlify so they will not exist when this check runs.